### PR TITLE
Move Default VB Version to 16.9

### PR DESCRIPTION
**Pull request by 333fred**
Fixes https://github.com/dotnet/roslyn/issues/57093.

### CODE REVIEW

**Comment by 333fred** [comment]
@dotnet/roslyn-compiler for an extremely small review.

**Review by Youssef1313 — COMMENTED** [review]
Minor: Can we document the enum members in a similar manner like C#?

https://github.com/dotnet/roslyn/blob/9aac29d81ed13fe2d704e45d18de44309be30730/src/Compilers/CSharp/Portable/LanguageVersion.cs#L220-L228

Also we need a doc update for [Select the Visual Basic language version](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version) article.

**Comment by jcouv** [comment]
There should be a test affected by this change.

**Comment by 333fred** [comment]
> There should be a test affected by this change.

I don't think this is worth doing.

---
In reply to: [966468306](https://github.com/dotnet/roslyn/pull/57688#issuecomment-966468306) [](http://example.com/codeflow?ancestors=966468306,966468306)

**Comment by 333fred** [comment]
@jcouv do you have any further feedback?

**Comment by 333fred** [comment]
@jcouv do you have any further feedback?

**Comment by jcouv** [comment]
I still think we should have a test. We generally do for public APIs, notwithstanding how simple they might be.
That said, I don't think it's worth spending this much time arguing for adding a 2 minutes test.

**Review by jcouv — APPROVED** [review]

**Comment by Youssef1313** [comment]
In case a test was decided to be written, there is an existing test for this API. No need for a completely new test.

https://github.com/dotnet/roslyn/blob/c7521c9ec5dbdc06778606960fe211c26420d826/src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb#L2063-L2078

**Comment by 333fred** [comment]
@dotnet/roslyn-compiler for a second review.

File: src/Compilers/VisualBasic/Portable/LanguageVersion.vb
Comment (Youssef1313) [inline src/Compilers/VisualBasic/Portable/LanguageVersion.vb:91]:
```suggestion
                Case LanguageVersion.Latest,
                     LanguageVersion.Default
```

Worth grouping both cases together so that future updates don't miss one of them?

Comment (333fred) [reply src/Compilers/VisualBasic/Portable/LanguageVersion.vb:91]: I'd prefer not. If we then need to update `Latest` independently again, it's a bigger change.

**Review by AlekseyTs — APPROVED** [review]
LGTM (commit 1)

File: src/Compilers/VisualBasic/Portable/LanguageVersion.vb
Comment (Youssef1313) [reply src/Compilers/VisualBasic/Portable/LanguageVersion.vb:91]: @333fred I thought the goal of the change is to keep "default" to **always** be "latest". If that's not the case, I'm interested to understand more on why the change is needed. Specially that the [current documentation](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version) mentions:

> The special strings default and latest resolve to the latest major and minor language versions installed on the build machine, respectively.

The latest major is `16.0`, and the latest minor is `16.9`. I was actually going to open a doc PR after this is merged to update this sentence to:

> The special strings default and latest resolve to the latest language version installed on the build machine.

**Pull request merged**
